scene/gui/texture_progress.cpp

```diff
@@ -224,90 +226,92 @@ void TextureProgress::_notification(int p_what) {
-							draw_texture_rect_region(progress, region, region);
-						} break;
-						case FILL_RIGHT_TO_LEFT: {
-							Rect2 region = Rect2(Point2(s.x - s.x * get_as_ratio(), 0), Size2(s.x * get_as_ratio(), s.y));
-							draw_texture_rect_region(progress, region, region);
-						} break;
-						case FILL_TOP_TO_BOTTOM: {
-							Rect2 region = Rect2(Point2(), Size2(s.x, s.y * get_as_ratio()));
-							draw_texture_rect_region(progress, region, region);
-						} break;
-						case FILL_BOTTOM_TO_TOP: {
-							Rect2 region = Rect2(Point2(0, s.y - s.y * get_as_ratio()), Size2(s.x, s.y * get_as_ratio()));
-							draw_texture_rect_region(progress, region, region);
-						} break;
-						case FILL_CLOCKWISE:
-						case FILL_COUNTER_CLOCKWISE: {
-							float val = get_as_ratio() * rad_max_degrees / 360;
-							if (val == 1) {
-								Rect2 region = Rect2(Point2(), s);
-								draw_texture_rect_region(progress, region, region);
-							} else if (val != 0) {
-								Array pts;
-								float direction = mode == FILL_CLOCKWISE ? 1 : -1;
-								float start = rad_init_angle / 360;
-								float end = start + direction * val;
-								pts.append(start);
-								pts.append(end);
-								float from = MIN(start, end);
-								float to = MAX(start, end);
-								for (int i = 0; i < 12; i++)
-									if (corners[i] > from && corners[i] < to)
-										pts.append(corners[i]);
-								pts.sort();
-								Vector<Point2> uvs;
-								Vector<Point2> points;
-								uvs.push_back(get_relative_center());
-								points.push_back(Point2(s.x * get_relative_center().x, s.y * get_relative_center().y));
-								for (int i = 0; i < pts.size(); i++) {
-									Point2 uv = unit_val_to_uv(pts[i]);
-									if (uvs.find(uv) >= 0)
-										continue;
-									uvs.push_back(uv);
-									points.push_back(Point2(uv.x * s.x, uv.y * s.y));
-								}
-								draw_polygon(points, Vector<Color>(), uvs, progress);
-							}
-							if (Engine::get_singleton()->is_editor_hint()) {
-								Point2 p = progress->get_size();
-								p.x *= get_relative_center().x;
-								p.y *= get_relative_center().y;
-								p = p.floor();
-								draw_line(p - Point2(8, 0), p + Point2(8, 0), Color(0.9, 0.5, 0.5), 2);
-								draw_line(p - Point2(0, 8), p + Point2(0, 8), Color(0.9, 0.5, 0.5), 2);
-							}
-						} break;
-						default:
-							draw_texture_rect_region(progress, Rect2(Point2(), Size2(s.x * get_as_ratio(), s.y)), Rect2(Point2(), Size2(s.x * get_as_ratio(), s.y)));
+		if (nine_patch_stretch && (mode == FILL_LEFT_TO_RIGHT || mode == FILL_RIGHT_TO_LEFT || mode == FILL_TOP_TO_BOTTOM || mode == FILL_BOTTOM_TO_TOP)) {
+			if (under.is_valid()) {
+				draw_nine_patch_stretched(under, FILL_LEFT_TO_RIGHT, 1.0);
+			}
+			if (progress.is_valid()) {
+				draw_nine_patch_stretched(progress, mode, get_as_ratio());
+			}
+			if (over.is_valid()) {
+				draw_nine_patch_stretched(over, FILL_LEFT_TO_RIGHT, 1.0);
+			}
+		}
+		else {
```

Comment:
Same here, incorrect style.

Reply:
Thanks. Sorry, I did not have the time to update this. I have the tool for formatting now installed so it should not happen in the future (Visual Studio automatically changed the switch statement formatting).